### ISSUE TITLE
Chronic duodenitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6191]
**Preferred term label:**
Chronic duodenitis

**Synonyms**
Chronic duodenal inflammation

**Definition (free text, please give PubMed ID)**
Chronic inflammation of the lining of the upper small intestine (duodenum).
PMID: 5829215

**Parent term (use hpo.jax.org/app)**
Duodenitis

**Comment by pnrobinson** [comment]
If there is a need to characterise a phenotypic feature as chronic, the HPO would usually use this modifier term
https://hpo.jax.org/app/browse/term/HP:0011010
Sometimes there is a dedicate "chronic" version of a term, usually this is if the distinction is essential for some differential diagnosis.
